src/main/java/com/github/onsdigital/logging/layouts/model/JsonLogItem.java

```diff
@@ -69,7 +70,7 @@ public JsonLogItem(ILoggingEvent event) {
 
     public String asJson() {
         try {
-            return new ObjectMapper().writeValueAsString(this);
+            return Beautifier.beautifyJson(this.event, new ObjectMapper().writeValueAsString(this));
```

Comment:
I was wondering if we should just have a static instance of ObjectMapper instead of creating a new one each time. Seems like its recommended but maybe worth double checking. (I see that it was already like this and not changed in your PR) https://stackoverflow.com/questions/3907929/should-i-declare-jacksons-objectmapper-as-a-static-field